File: Data Structures & Algorithms/duplicate-integer/submission-0.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        lst = []
        i = 0
        while len(lst) != len(nums):
            if nums[i] not in lst:
                lst.append(nums[i])
            else:
                return True
            i+=1
        return False

        

File: Data Structures & Algorithms/two-integer-sum/submission-0.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {

        int[] res = {-1,-1};
        Dictionary<int, int> dict = new Dictionary<int, int>();

        for (int i = 0; i < nums.Length; i++){
            int rest = target - nums[i];

            if (dict.ContainsKey(rest)){
                res[0] = dict[rest];
                res[1] = i;
                return res;
            }
            else{
                dict[nums[i]] = i;
            }

        }
        return res;


    }
}
